employee-rostering-server/src/main/filtered-resources/META-INF/persistence.xml

```diff
@@ -4,6 +4,7 @@
   <persistence-unit name="optaweb-employee-rostering-persistence-unit" transaction-type="JTA">
     <description>OptaWeb Employee Rostering Persistence Unit</description>
     <jta-data-source>${org.optaweb.employeerostering.persistence.datasource}</jta-data-source>
+    <mapping-file>META-INF/orm.xml</mapping-file>
```

Comment:
Check if the benchmark app still works - that has a copy of this persistence.xml without filtering

Reply:
@ge0ffrey The benchmark app still worked (checked by running `mvn org.codehaus.mojo:exec-maven-plugin:1.6.0:java -Dexec.mainClass=org.optaweb.employeerostering.benchmark.OptaWebEmployeeRosteringBenchmarkApp`)